javascript/builtins/Array.json

```diff
@@ -836,10 +836,10 @@
                 "version_added": true
               },
               "firefox": {
-                "version_added": true
+                "version_added": "1.5"
```

Comment:
Likely correct, but can you double check that you indeed wanted 1.5 there. It is not listed in http://www.squarefree.com/burningedge/releases/1.5-comprehensive.html .

Reply:
This is like `indexOf` and see also https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/1.6